BiweeklyMeetings/2022-08-16.md

```diff
@@ -0,0 +1,406 @@
+
+**Andrew** -: Yeah. So they're actually one of the people that is involved with this. So maybe if we can get them to comment on it, I'll talk to them about that.
+
+**Matthias** -: And this is something I would love to see in the rationale section as well. Exactly what you just explained. This is a collaborative effort between actual people building with, in the music industry, on, kind of, i don't know, the blockchain in general and that's also why we justifying some of the choices that have been made.
+**Andrew** -: Okay.
+
+**Matthias** -: That just gives a bit of context on where the decision comes from in the legitivity of that.
+
+**Andrew** -: Okay.
+
+**Matthias** -: From a technical standpoint, just to clarify things, this is proposing to extend the on chain with the data and the set of fields that is provided alongside current NFTs from the CIP 25 right?
+
+**Andrew** -: That's correct. It supports CIP 25 version one and version two.
+
+**Matthias** -: Okay. Then should we maybe move it to last check for the next goal provided that this was already discussed with a few?
+
+**Andrew** -: Yeah. We've had several two hour meetings with people. There was 20 different groups in the room, either producing music on Cardano or representing wallets or players or things like that. So yeah, I will add to the rationale section before next meeting that this comes up.
+
+**Matthias** -: Okay. Thanks.
+
+**Andrew** -: Probably not next meeting, but maybe the meeting after, because the next meeting is always in the times I can't be at.
+
+**Matthias** -: You're right. So let's keep that in for the US based next meeting.
+
+**Andrew** -: Yep, thank you.
+
+
+### Issues
+N/A
+
+### Close 
+
+**Matthias** -: So let's just wrap up quickly on what we just discussed. We've been through CIP 51 at the beginning, which we agreed to rediscuss, bring back on the table after [inaudible 00:48:13] happened. Since this anyway, won't happen before [inaudible 00:48:19] and further discussions are needed, then CIP 48 extended submitted data for, sorry, for CIP 25. CIP is a bit rushed out in terms of quality and format.
+So the content itself is okay technically-wise, but there are quite a few shortcomings when it comes to how it is written. So we ask the author to fix that. CIP 30, we have agreed to just close up requests. It's been inactive for a while. The author is not responsive, so I can't close it, unfortunately, because I'm on windows. But if any of the other auditors want to do it, please do. CIP 45 then the using pledge as a bidding program, there is some overlap with CIP 50. This is part of the broader discussions about keywords and incentives and there were concerns being raised about this actually incentivizing large pledge owners to just create [inaudible 00:49:30] without actually impacting, that is a tradition of Cardano. At least not in a good way.
```

Comment:
```suggestion
So the content itself is okay technically-wise, but there are quite a few shortcomings when it comes to how it is written. So we asked the author to fix that. CIP 30, we have agreed to just close up requests. It's been inactive for a while. The author is not responsive. I can't close it myself, unfortunately, because I'm on windows. But if any of the other auditors want to do it, please do. CIP 45 then the using pledge as a bidding program, there is some overlap with CIP 50. This is part of the broader discussions about rewards and incentives and there were concerns being raised about this actually incentivizing large pledge owners to just create more stake pools without actually impacting the decentralization of Cardano. At least not in a good way.
```